src/App.tsx

```diff
@@ -9,39 +9,30 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
-import { User } from './types/User';
-import { Post } from './types/Post';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { fetchUsers } from './features/userSlice';
+import { fetchPosts, postsSelector } from './features/postSlice';
+import { authorSelector } from './features/authorSlice';
+import { selectedPostSelector } from './features/selectedPostSlice';
+import { set } from './features/selectedPostSlice';
```

Comment:
also use set as name in this place, change this naming. 
We can set different data